### PR TITLE
Added the additional fields according to datev V.5.0

### PR DESCRIPTION
**Pull request by mseel**

### CODE REVIEW

File: lib/datev/export.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def <<(attributes)
     def to_s
       CSV.generate(CSV_OPTIONS) do |csv|
         write(csv)
-      end
+      end.encode(Encoding::WINDOWS_1252)
```
Comment (ledermann) [inline lib/datev/export.rb]: Why do we need this? Specs are running fine on Linux (at Travis) and my local OS X machine without this change. Encoding is already set to `windows-1252` by the `CSV_OPTIONS` hash.

Comment (mseel) [inline lib/datev/export.rb]: i had an issue with encoding on ruby 2.0 and OS X - shall i remove it?

Comment (ledermann) [inline lib/datev/export.rb]: I've just run the specs with Ruby 2.0.0p648 on OS X - no problems with AND without your change. So, until we are not sure that the change is really needed I prefer to remove it.

**Comment by ledermann** [comment]
Thanks, @mseel!

**Pull request merged**